src/libse/Dictionaries/NameList.cs

```diff
@@ -258,13 +258,25 @@ public bool Add(string name)
             }
 
             var de = nameListXml.DocumentElement;
-            if (de != null)
+            if (de == null) 
+            { 
+                return true; 
+            }
+
+            var node = nameListXml.CreateElement("name");
+            node.InnerText = name;
+            de.AppendChild(node);
+            if(LanguageName == "en")
             {
-                var node = nameListXml.CreateElement("name");
-                node.InnerText = name;
-                de.AppendChild(node);
-                nameListXml.Save(fileName);
+                if(!(name.EndsWith("'s") || name.EndsWith("'")))
```

Comment:
I don't think this is a good idea it will only increase the name file size
This type of process can easily be handled in code

Reply:
Yes, I've been thinking about this on and off for a while, 
this is definitely _not the correct solution_ but at the time I thought it was helpful.
I meant to take it down but forgot.
It also doesn't account for cases like, 
_"John'll be back soon."_

Using subedit regularly for the past few months has starkly pointed out the issues with name handling.
Like how names can't have spaces in them.
_"Zhu Li, do the thing!" - Varrick, The Legend of Korra_

Reply:
Or how you need to create multiple ocr fixes to account for errors in case.
_Zhu li -> Zhu Li
zhu Li -> Zhu Li
zhu li -> Zhu Li_

Reply:
It would also be nice to be able to have a generic name keyword. 
Example:
Find and Replace:
[Name]'ll
With: 
[Name] will

or

[Name]
with:
\<i>[Name]\</i>

idk ¯\_(ツ)_/¯ like you said, achievable through code but not everyone is a coder.